### PR TITLE
Add bind vars to AQL query UI

### LINKED ISSUE #119: Change the "clear" button to a "clear results" button in the AQL UI
Currently the clear button clears the query, which was immediately apparent to me. ~We should either change the wording to reflect that it just clears the query text or we could update the functionality to clear out the query results, too.~

TODO:
- [x] change the button text to `clear results`
- [x] change its behavior to do so

### PR DESCRIPTION
**Pull request by AlmightyYakob**
Client change for https://github.com/multinet-app/multinet-api/pull/139

Also closes #119

### CODE REVIEW

File: src/views/AQLWizard.vue [hunk 1]
```diff
@@ -71,36 +113,6 @@
...
-                label="Table Name"
-                append-icon="publish"
-                :error-messages="createTableErrorMessage"
-                :hide-details="!createTableErrorMessage"
-                @input="createTableErrorMessage = null"
-                @click:append="createTable"
-                @keydown.enter="createTable"
-              />
-            </v-card>
-          </v-menu>
```
Comment (JackWilb) [inline src/views/AQLWizard.vue:103]: Why remove this? It's a cool feature of this view

**Review by JackWilb — APPROVED** [review]
Looks good. I had one comment about some removed functionality.

Also, we will need to document @@ vs @ (or at least provide a link to arango docs). We should make an issue for that

File: src/views/AQLWizard.vue (hunk 1 above)
Comment (AlmightyYakob) [reply src/views/AQLWizard.vue:103]: It doesn't actually work (there is no API endpoint that allows creation of a table from an AQL query), and adding it is out of scope for this PR. 

I don't know when the feature was removed / lost from the API, it obviously used to work though.

Comment (JackWilb) [reply src/views/AQLWizard.vue:103]: Gotcha. Makes sense. Happy to merge as is then

**Review by JackWilb — APPROVED** [review]

**Comment by AlmightyYakob** [comment]
@JackWilb Can I get a re-review since updating the client library.

**Review by JackWilb — APPROVED** [review]

**Pull request merged**